File: Konfigurator-nestjs-Backend/konfigurator-nestjs-backend/src/dto/ServiceResponse.dto.ts
export class ServiceResponseDto<T> {
    constructor(
        data: T,
        success: boolean,
        message: string
    ) {
        this.data = data;
        this.success = success;
        this.message = message;
    }
    
    data: T;
    success: boolean;
    message: string;
}

File: Konfigurator-aspnetcore-Backend/Konfigurator-aspnetcore-Backend/Models/DTOs/ServiceResponseDto.cs
using System;
namespace Konfigurator_aspnetcore_Backend.Models.DTOs
{
	public class ServiceResponseDto<T>
	{
		public T? Data { get; set; }
		public bool Success { get; set; } = true;
		public string Message { get; set; } = string.Empty;
	}

	public class ServiceResponse
	{
		public bool Success { get; set; } = true;
		public string Message { get; set; } = string.Empty;
	}
}

